website/src/controllers/api-v3/members.js

```diff
@@ -231,4 +241,76 @@ api.getChallengeMemberProgress = {
   },
 };
 
+async function sendMessage (sender, userToReceiveMessage, messageData) {
```

Comment:
Yea, that sounds like a good place for it now. It might be good as a Mail micro-service in the future.